plugins/inputs/monit/monit_test.go

```diff
@@ -0,0 +1,208 @@
+package monit
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMonitAll(t *testing.T) {
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
[no change required] Personally, I would have created one `NewServer` so these tests could be run concurrently, this appears to be working though